### PR TITLE
feat: Created custom dashboard in cloud monitoring to web_app_protection_example 

### PR DESCRIPTION
**Pull request by mbuitragos**
## What?

* Adding a custom dashboard to show information about the resources. [CIT-183]

## Why?

Custom dashboard created in Cloud Monitoring.

## Testing ?

Integration and Lint tests were performed.

## Anything Else?

None.

### CODE REVIEW

File: examples/web_app_protection_example/main.tf [hunk 1]
```diff
@@ -505,3 +505,51 @@ module "lb-http" {
...
+
+    labels {
+      key        = "signature_id"
+      value_type = "STRING"
+    }
+  }
+  label_extractors = {
+    "signature_id" = "EXTRACT(jsonPayload.enforcedSecurityPolicy.preconfiguredExprIds)"
+  }
+}
```
Comment (shakeebshams) [inline examples/web_app_protection_example/main.tf:555]: needs newline at end of file

File: examples/web_app_protection_example/scripts/dashboard.json [hunk 2]
```diff
@@ -0,0 +1,755 @@
...
+            }
+          }
+        },
+        "width": 6,
+        "xPos": 6,
+        "yPos": 11
+      }
+    ]
+  }
+}
```
Comment (shakeebshams) [inline examples/web_app_protection_example/scripts/dashboard.json:755]: newline here as well

File: examples/web_app_protection_example/main.tf (hunk 1 above)
Comment (mbuitragos) [reply examples/web_app_protection_example/main.tf:555]: The file already has a line at the end, github does not show it, only downloading the repository to validate.

File: examples/web_app_protection_example/scripts/dashboard.json (hunk 2 above)
Comment (mbuitragos) [reply examples/web_app_protection_example/scripts/dashboard.json:755]: The file already has a line at the end, github does not show it, only downloading the repository to validate.

**Review by gtsorbo — APPROVED** [review]

**Pull request merged**